### PR TITLE
Feature/fulldialog

### PR DESCRIPTION
**Pull request by Blogbotana**

### CODE REVIEW

File: TelegramBot/TelegramBot/CallbackHandler.cs [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+        }
+
+        private async Task ReplyToUserButtonsMainMenu(CallbackQuery query, Message lastmessage, string button)
+        {
+            string enum1 = "UserButtonsMainMenu.";
+            switch (button)
+            {
+                case string result when result == enum1 + "Support":
+                    {
+                        //await TGBot.MyBot.BotClient.DeleteMessageAsync(query.From.Id, lastmessage.MessageId, TGBot.MyBot.CancellToket);
```
Comment (destros52) [inline TelegramBot/TelegramBot/CallbackHandler.cs:62]: remove

File: TelegramBot/TelegramBot/DialogFunction.cs [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+        }
+
+        private IReplyMarkup HideKeyBoard()
+        {
+            return new ReplyKeyboardRemove();
+        }
+
+        private InlineKeyboardMarkup GetButtonsMainMenu()
+        {
+            string enum1 = "UserButtonsMainMenu.";
```
Comment (destros52) [inline TelegramBot/TelegramBot/DialogFunction.cs:86]: magic word

File: TelegramBot/TelegramBot/TGBot.cs [hunk 3]
```diff
@@ -3,48 +3,86 @@
...
-                return;
-            // Only process text messages
-            if (update.Message!.Type != MessageType.Text)
-                return;
-            
-            //var chatId = update.Message.Chat.Id;
-            //var messageText = update.Message.Text;
+            Task? handler = update.Type switch
+            {
+                // UpdateType.Unknown:
```
Comment (destros52) [inline TelegramBot/TelegramBot/TGBot.cs:61]: remove

File: TelegramBot/TelegramBot/TGBot.cs [hunk 4]
```diff
@@ -3,48 +3,86 @@
...
+                // UpdateType.ChannelPost:
```
Comment (destros52) [inline TelegramBot/TelegramBot/TGBot.cs:62]: remove

File: TelegramBot/TelegramBot/DialogFunction.cs (hunk 2 above)
Comment (destros52) [inline TelegramBot/TelegramBot/DialogFunction.cs]: still here

**Review by destros52 — CHANGES_REQUESTED** [review]

File: TelegramBot/TelegramBot/DialogFunction.cs (hunk 2 above)
Comment (Blogbotana) [reply TelegramBot/TelegramBot/DialogFunction.cs]: image

**Review by destros52 — APPROVED** [review]

**Pull request merged**